src/tag/tag.js

```diff
@@ -13,13 +13,12 @@ import {
   Root as StyledRoot,
   ActionIcon as StyledActionIcon,
 } from './styled-components';
-import {STYLE} from './constants';
 
 class Tag extends React.Component<PropsT, {}> {
   static defaultProps = {
     overrides: {},
     onActionClick: () => {},
-    color: STYLE.primary,
+    color: undefined,
```

Comment:
we should still set the default value to be the `COLOR_STYLE_KEYS.primary`

Reply:
If we overload color to take theme keys, then yes, but that seems potentially problematic. At a minimum, if we took that approach, we'd want to add checks to ensure that we never had namespace conflicts between the theme color values and css named colors.

Reply:
That's a good point.

Reply:
Do we even need to set this default, then?